### ISSUE TITLE
Cicero CLI does not render multi-line, unordered list when --unquoteVariables true.

### ISSUE THREAD

**Issue by martinhalford** [accordproject/cicero#561]
**Describe the bug**
When rendering a `draft` markdown containing an unordered list `{{#ulist}}` split across multiple lines, and with the command line parameter `--unquoteVariables` equal to `true`, the output markdown for the `{{#ulist}}` is not rendered.

**To Reproduce**
1. Add a `{{#ulist}}` to the `grammar.tem.md` file. 

For example:
```
{{#ulist vendors}}_Name:_ **{{fullName}}**
_Email:_ **{{email}}**
_Mobile:_ **{{mobile}}**
_Phone:_  **{{phone}}**
_ABN:_  **{{ABN}}**
_ACN:_  **{{ACN}}**

{{/ulist}}
```
using `data.json` such as....
```
  "vendors": [
      {
        "$class": "systems.benext.reiv.Person",
        "partyId": "7085145d-f5e8-4f3e-ac5d-d0cb002f9324",
        "fullName": "Jack Walnut",
        "email": "jackwalnut@gmail.com",
        "mobile": "0432 123456",
        "phone": "(03) 887 7766",
        "ABN": "Not applicable",
        "ACN": "Not applicable"
    },
    {
      "$class": "systems.benext.reiv.Person",
      "partyId": "7085145d-f5e8-4f3e-ac5d-d1cb001f9314",
      "fullName": "Betty Beetroot",
      "email": "bettybeetroot@gmail.com",
      "mobile": "0422 4545454",
      "phone": "(03) 996 6677",
      "ABN": "Not applicable",
      "ACN": "Not applicable"
  }
  ],
```

2. Execute `draft` command with `--unquoteVariables false`.

For example:
`cicero draft --template . --data ./text/data.json --output ./text/sample.md --unquoteVariables false`

3. View `sample.md` file and should see markdown like this...

```
-  *Name:* **"Jack Walnut"**,
   *Email:* **"jackwalnut@gmail.com"**,
   *Mobile:* **"0432 123456"**,
   *Phone:*  **"(03) 887 7766"**,
   *ABN:*  **"Not applicable"**,
   *ACN:*  **"Not applicable"**

-  *Name:* **"Betty Beetroot"**,
   *Email:* **"bettybeetroot@gmail.com"**,
   *Mobile:* **"0422 4545454"**,
   *Phone:*  **"(03) 996 6677"**,
   *ABN:*  **"Not applicable"**,
   *ACN:*  **"Not applicable"**
```
which, when seen formatted in markdown viewer, renders as:

-  *Name:* **"Jack Walnut"**,
   *Email:* **"jackwalnut@gmail.com"**,
   *Mobile:* **"0432 123456"**,
   *Phone:*  **"(03) 887 7766"**,
   *ABN:*  **"Not applicable"**,
   *ACN:*  **"Not applicable"**

-  *Name:* **"Betty Beetroot"**,
   *Email:* **"bettybeetroot@gmail.com"**,
   *Mobile:* **"0422 4545454"**,
   *Phone:*  **"(03) 996 6677"**,
   *ABN:*  **"Not applicable"**,
   *ACN:*  **"Not applicable"**

4. Now execute `draft` command with `--unquoteVariables true`.

For example:
`cicero draft --template . --data ./text/data.json --output ./text/sample.md --unquoteVariables true`

5. View `sample.md` file and should see markdown like this...

``` 
```<list/>
-  *Name:* **Jack Walnut**,
   *Email:* **jackwalnut@gmail.com**,
   *Mobile:* **0432 123456**,
   *Phone:*  **(03) 887 7766**,
   *ABN:*  **Not applicable**,
   *ACN:*  **Not applicable**

-  *Name:* **Betty Beetroot**,
   *Email:* **bettybeetroot@gmail.com**,
   *Mobile:* **0422 4545454**,
   *Phone:*  **(03) 996 6677**,
   *ABN:*  **Not applicable**,
   *ACN:*  **Not applicable**```
```
which, when seen formatted in markdown viewer, renders as:

```
<list/>
-  *Name:* **Jack Walnut**,
   *Email:* **jackwalnut@gmail.com**,
   *Mobile:* **0432 123456**,
   *Phone:*  **(03) 887 7766**,
   *ABN:*  **Not applicable**,
   *ACN:*  **Not applicable**

-  *Name:* **Betty Beetroot**,
   *Email:* **bettybeetroot@gmail.com**,
   *Mobile:* **0422 4545454**,
   *Phone:*  **(03) 996 6677**,
   *ABN:*  **Not applicable**,
   *ACN:*  **Not applicable**
```

**Expected behavior**

Expect sample.md file to contain same markdown for `--unquoteVariables true` and `--unquoteVariables false` except with quotes missing or present respectively, like this...

```
-  *Name:* **Jack Walnut**,
   *Email:* **jackwalnut@gmail.com**,
   *Mobile:* **0432 123456**,
   *Phone:*  **(03) 887 7766**,
   *ABN:*  **Not applicable**,
   *ACN:*  **Not applicable**

-  *Name:* **Betty Beetroot**,
   *Email:* **bettybeetroot@gmail.com**,
   *Mobile:* **0422 4545454**,
   *Phone:*  **(03) 996 6677**,
   *ABN:*  **Not applicable**,
   *ACN:*  **Not applicable**
```
which, when seen formatted in markdown viewer, renders as:

-  *Name:* **Jack Walnut**,
   *Email:* **jackwalnut@gmail.com**,
   *Mobile:* **0432 123456**,
   *Phone:*  **(03) 887 7766**,
   *ABN:*  **Not applicable**,
   *ACN:*  **Not applicable**

-  *Name:* **Betty Beetroot**,
   *Email:* **bettybeetroot@gmail.com**,
   *Mobile:* **0422 4545454**,
   *Phone:*  **(03) 996 6677**,
   *ABN:*  **Not applicable**,
   *ACN:*  **Not applicable**

**Cicero:**
 - Version 0.20.10

**Comment by jeromesimeon** [comment]
hi @martinhalford Thanks for the report. There is quite a lot of different things to unpack from this one example. But I thought I would at least report on the `unquoteVariables` option part. The option doesn't take `true/false` you just set it. I think it works as expected in this example:

````
bash-3.2$ cat text/grammar.tem.md 
This is a contract

{{#ulist vendors}}_Name:_ **{{fullName}}**
_Email:_ **{{email}}**
_Mobile:_ **{{mobile}}**
_Phone:_  **{{phone}}**
_ABN:_  **{{ABN}}**
_ACN:_  **{{ACN}}**
{{/ulist}}

bash-3.2$ cat data.json 
{ "$class" : "org.accordproject.listquote.VendorsContract",
  "contractId" : "12345",
  "vendors" : [
      { "$class" : "org.accordproject.listquote.Vendor",
        "fullName" : "Jack Walnut",
        "email" : "jackwalnut@gmail.com",
        "mobile" : "0432 123456",
        "phone" : "(03) 887 7766",
        "ABN" : "Not applicable",
        "ACN" : "Not applicable" },
      { "$class" : "org.accordproject.listquote.Vendor",
        "fullName" : "Betty Beetroot",
        "email" : "bettybeetroot@gmail.com",
        "mobile" : "0422 4545454",
        "phone" : "(03) 996 6677",
        "ABN" : "Not applicable",
        "ACN" : "Not applicable" }
  ] }
bash-3.2$ cicero draft --unquoteVariables
1:30:57 PM - info: Using current directory as template folder
1:30:57 PM - info: Loading a default data.json file.
1:30:58 PM - info: This is a contract

``` <list/>
-  *Name:* **Jack Walnut**
   *Email:* **jackwalnut@gmail.com**
   *Mobile:* **0432 123456**
   *Phone:*  **(03) 887 7766**
   *ABN:*  **Not applicable**
   *ACN:*  **Not applicable**

-  *Name:* **Betty Beetroot**
   *Email:* **bettybeetroot@gmail.com**
   *Mobile:* **0422 4545454**
   *Phone:*  **(03) 996 6677**
   *ABN:*  **Not applicable**
   *ACN:*  **Not applicable**
```

This is more text
````

**Comment by martinhalford** [comment]
Hi @jeromesimeon, Thanks for looking into this.  

Interesting you say that the `unquoteVariables` option "doesn't take true/false you just set it."  

If this is the case, then the `cicero draft --help` might need to be updated, as it states that `--unquoteVariables` is a boolean with the implication it can be set to `true` or `false`.
```
  --unquoteVariables  remove variables quoting  [boolean] [default: false]
```

When `--unquoteVariables` is omitted (or explicitly set to `false`) you get the following:
```
cicero draft --unquoteVariables false
3:17:14 PM - info: Using current directory as template folder
3:17:14 PM - info: Loading a default data.json file.
3:17:16 PM - info: This is a contract

-  *Name:* **"Jack Walnut"**
   *Email:* **"jackwalnut@gmail.com"**
   *Mobile:* **"0432 123456"**
   *Phone:*  **"(03) 887 7766"**
   *ABN:*  **"Not applicable"**
   *ACN:*  **"Not applicable"**

-  *Name:* **"Betty Beetroot"**
   *Email:* **"bettybeetroot@gmail.com"**
   *Mobile:* **"0422 4545454"**
   *Phone:*  **"(03) 996 6677"**
   *ABN:*  **"Not applicable"**
   *ACN:*  **"Not applicable"**

This is more text
``` 
The above ***IS*** what I was expecting for `--unquoteVariables false`.

When `--unquoteVariables` is included (or explicitly set to `true`) you get the following:

```
$ cicero draft --unquoteVariables true
3:25:43 PM - info: Using current directory as template folder
3:25:43 PM - info: Loading a default data.json file.
3:25:44 PM - info: This is a contract

```<list/>
-  *Name:* **Jack Walnut**
   *Email:* **jackwalnut@gmail.com**
   *Mobile:* **0432 123456**
   *Phone:*  **(03) 887 7766**
   *ABN:*  **Not applicable**
   *ACN:*  **Not applicable**

-  *Name:* **Betty Beetroot**
   *Email:* **bettybeetroot@gmail.com**
   *Mobile:* **0422 4545454**
   *Phone:*  **(03) 996 6677**
   *ABN:*  **Not applicable**
   *ACN:*  **Not applicable**
` ```

This is more text
```

The above is ***NOT*** what I was expecting - or hoping to see.  

Below is what I was expecting and hoping to see:

```
cicero draft --unquoteVariables true
3:17:14 PM - info: Using current directory as template folder
3:17:14 PM - info: Loading a default data.json file.
3:17:16 PM - info: This is a contract

-  *Name:* **Jack Walnut**
   *Email:* **jackwalnut@gmail.com**
   *Mobile:* **0432 123456**
   *Phone:*  **(03) 887 7766**
   *ABN:*  **Not applicable**
   *ACN:*  **Not applicable**

-  *Name:* **Betty Beetroot**
   *Email:* **bettybeetroot@gmail.com**
   *Mobile:* **0422 4545454**
   *Phone:*  **(03) 996 6677**
   *ABN:*  **Not applicable**
   *ACN:*  **Not applicable**

This is more text
``` 
So... as far as I can tell, the `true` & `false` flags work the same as if `--unquoteVariables` is explicitly set or omitted.  

Bottom line, I do not think anyone wants the see the output rendered as a code `<list>`.

If the `cicero draft --unquoteVarables` could simply create output without the quotes then that would be super awesome!

Please let me know your thoughts.

**Comment by jeromesimeon** [comment]
> If the `cicero draft --unquoteVarables` could simply create output without the quotes then that would be super awesome!
> 

I believe it does!

**Comment by martinhalford** [comment]
Apologies @jeromesimeon 
- Yes, the `cicero draft --unquoteVariables` command does render the output in an unordered list but only inside a code block with a `<list/>` tag, which does not render properly in a markdown document.

This is what the request looks like when run from Cicero-Server with `"unquoteVariables": true`.

![image](https://user-images.githubusercontent.com/970336/88365657-1ce3cb00-cdca-11ea-88ad-862f2079ce61.png)

However, works perfectly when `"unquoteVariables": false`.

![image](https://user-images.githubusercontent.com/970336/88365613-f887ee80-cdc9-11ea-974a-68197d04cb30.png)

Here's the curl to the `ListQuote` contract hosted on our dev server.

```
curl --request POST \
  --url http://cicero.benext.systems:6001/draft/listquote \
  --header 'accept: application/json' \
  --header 'content-type: application/json' \
  --data '{
		"data": { "$class" : "org.accordproject.listquote.VendorsContract",
		"contractId" : "12345",
		"vendors" : [
				{ "$class" : "org.accordproject.listquote.Vendor",
					"fullName" : "Jack Walnut",
					"email" : "jackwalnut@gmail.com",
					"mobile" : "0432 123456",
					"phone" : "(03) 887 7766",
					"ABN" : "Not applicable",
					"ACN" : "Not applicable" },
				{ "$class" : "org.accordproject.listquote.Vendor",
					"fullName" : "Betty Beetroot",
					"email" : "bettybeetroot@gmail.com",
					"mobile" : "0422 4545454",
					"phone" : "(03) 996 6677",
					"ABN" : "Not applicable",
					"ACN" : "Not applicable" }
		] 
	},
		"options": {
			"unquoteVariables": true
		}
}'

```

**Comment by martinhalford** [comment]
Fixed in cicero version 0.21.0 👍 

![image](https://user-images.githubusercontent.com/970336/88854759-1f647b80-d235-11ea-8da2-4e1e232bb3cf.png)
